Add description of vshard.*.info() options (#3306)
tarantool/doc: doc/reference/reference_rock/vshard/vshard_router.rst

..  _router_api-info:

..  function:: vshard.router.info({options})

>     Return information about each instance. Since vshard v.0.1.22 the

Reviewer: ```suggestion
    Return information about each instance. Since vshard v.0.1.22, the
```

Author: Applied

    function also accepts options, which can be used to get additional
    information.

    :param options:

       *    ``with_services`` — a bool value. If set to ``true``, the
            function returns information about the background services

>             (such as discovery, master search or failover) that are

Reviewer: ```suggestion
            (such as discovery, master search, or failover) that are
```
Missing Oxford comma

>             working on the current instance.

Reviewer: ```suggestion
       *    ``with_services`` — a bool value. If set to ``true``, the
            function returns information about the background services
            (such as discovery, master search, or failover) that are
            working on the current instance.
```

Author: Applied

    * ``unavailable``—the number of buckets known to the ``router`` but unavailable for any requests
    * ``unreachable``—the number of buckets whose replica sets are not known to the ``router``

    Service parameters:

>     * ``name`` — the name of the service (``discovery``, ``failover``,
>       ``master_search``)
>     * ``status`` — status of the service(``ok``, ``error``)
>     * ``error`` — error message, which presents when ``status`` is ``error``.
>     * ``activity`` — state of the service. Shows, what it is doing right now
>       (e.g. ``updating replicas``).
>     * ``status_idx`` — incrementing counter of the status changes.
>       ``ok`` status is updated on every successful iteration of the service.
>       ``error`` is updated only when it is fixed.

Reviewer: ```suggestion
    * ``name`` — service name. Possible values: ``discovery``, ``failover``,
      ``master_search``.
    * ``status`` — service status. Possible values: ``ok``, ``error``.
    * ``error`` — error message that appears on the ``error`` status.
    * ``activity`` — service state. It shows what the service is currently doing
      (for example, ``updating replicas``).
    * ``status_idx`` — incrementing counter of the status changes.
      The ``ok`` status is updated on every successful iteration of the service.
      The ``error`` status is updated only when it is fixed.
```

Author: Applied
